src/digitalIdentity-frontend/src/app/components/navigation-bar/navigation-bar.component.ts

```diff
@@ -114,8 +114,10 @@ export class NavigationBarComponent implements OnInit {
   }
 
   async ngOnInit() {
+    // TODO: this is not a proper way to secure the app
```

Comment:
This is okay because all important data comes from the backend which is secured properly. 

If someone changes the sources here he will become 403 error requests... And will see e.g. an empty frontend table..

In my opinion that's completely acceptable and doesn't lower the security.